homework1/task02/Fibonacci_sequence.py

```diff
@@ -10,7 +10,9 @@
 
 def check_fibonacci(data: Sequence) -> bool:
     for i in range(len(data) - 2):
-        if data[i] + data[i + 1] == data[i + 2]:
+        if i < 0:
+            return False
+        if data[i] + data[i + 1] == data[i + 2] and data[i] != data[i + 1]:
             continue
         else:

```

Comment:
Here I's possible to avoid if .. else statement structure. 
consider this case::

    for (iterating through sequence):  
       if data[i] + data[i+1] != data[i +2]:  # checking only single case!
          return False
       if data[i] == data[i+1]:   # checking another case!
          return False
    return True   # no errors during iteration, hence the sequence is all good